bid.proto

```diff
@@ -22,7 +22,7 @@ message BidTransactionRequest {
     uint64 value = 2;
     bytes secret = 3; // JubJubCompressed
     StealthAddress stealth_address = 4;
-    bytes seed = 5; // BlsScalar
+    byte seed = 5; // u8
```

Comment:
The comment is redundant since you're sending a `byte` and not `bytes` (the comments where introduced to let a developer knows which kind of structure expects when `bytes` where sent).